workflows/GeneSeqToFamily/GeneSeqToFamily.ga

```diff
@@ -338,46 +457,41 @@
                 }
             ], 
             "position": {
-                "left": 1317.921875, 
-                "top": 207
+                "left": 1518.90625, 
+                "top": 392.5
             }, 
-            "post_job_actions": {
-                "HideDatasetActionoutputFile": {
-                    "action_arguments": {}, 
-                    "action_type": "HideDatasetAction", 
-                    "output_name": "outputFile"
-                }
-            }, 
-            "tool_id": "toolshed.g2.bx.psu.edu/repos/earlhaminst/hcluster_sg/hcluster_sg/0.5.1", 
+            "post_job_actions": {}, 
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/earlhaminst/hcluster_sg/hcluster_sg/0.5.1.1", 
             "tool_shed_repository": {
                 "changeset_revision": "1d2de27d7a9c", 
                 "name": "hcluster_sg", 
                 "owner": "earlhaminst", 
                 "tool_shed": "toolshed.g2.bx.psu.edu"
             }, 
-            "tool_state": "{\"__page__\": null, \"__rerun_remap_job_id__\": null, \"c_conditional\": \"{\\\"c\\\": \\\"\\\", \\\"b\\\": \\\"0.1\\\", \\\"m\\\": \\\"500\\\", \\\"C\\\": {\\\"__class__\\\": \\\"RuntimeValue\\\"}, \\\"O\\\": {\\\"O_selector\\\": \\\"\\\", \\\"__current_case__\\\": 1}, \\\"L\\\": \\\"2\\\", \\\"s\\\": \\\"0.34\\\", \\\"__current_case__\\\": 1}\", \"w\": \"\\\"0\\\"\", \"inputFile\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\"}", 
+            "tool_state": "{\"__page__\": null, \"__rerun_remap_job_id__\": null, \"c_conditional\": \"{\\\"C\\\": {\\\"__class__\\\": \\\"RuntimeValue\\\"}, \\\"L\\\": \\\"2\\\", \\\"O\\\": {\\\"O_selector\\\": \\\"\\\", \\\"__current_case__\\\": 1}, \\\"__current_case__\\\": 1, \\\"b\\\": \\\"0.1\\\", \\\"c\\\": \\\"\\\", \\\"m\\\": \\\"500\\\", \\\"s\\\": \\\"0.34\\\"}\", \"w\": \"\\\"0\\\"\", \"inputFile\": \"{\\\"__class__\\\": \\\"ConnectedValue\\\"}\"}", 
             "tool_version": "0.5.1.1", 
             "type": "tool", 
             "uuid": "d2b2ba89-b47d-4241-8adf-0d780f834412", 
-            "workflow_outputs": []
+            "workflow_outputs": [
+                {
+                    "label": null, 
+                    "output_name": "outputFile", 
+                    "uuid": "82fefa51-b192-49b2-bf41-e9914f179984"
+                }
+            ]
```

Comment:
The hcluster_sg output was not kept before, not sure if it's worth.

Reply:
It is good to be able to check clusters while the rest of the steps are running